client/src/components/Viewer/Viewer.js

```diff
@@ -7,11 +7,13 @@ import { config } from '../Config/config';
 import { toast } from 'react-toastify';
 import { io } from 'socket.io-client';
 import StatusInfo from '../statusInfo';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
 
 function Viewer() {
   const [currentFile, setCurrentFile] = useState({
     count: 0,
-    name: '',
+    names: '',
```

Comment:
this `name` variable is confusing. This is file and not filename. 
Please use `files` as variable name

Reply:
resolved

Reply:
resolved